Brick-Studios/BeastArena
# Animations fix

# Description

You can now actually create new animations xD
(Also added it to the systems so it can be used in pause)

# How can this be tested?
Test with https://github.com/Brick-Studios/BrickEngine/pull/66
Do you see the tree animating?
Does it stop animating when you pause?

# Checklist:
